example.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * a demo of gridualizer. Pulls down a grid checked into the repository, and allows you to change visualizations
+ */
+
+import React, { Component } from 'react'
+import { Map as LeafletMap, TileLayer } from 'react-leaflet'
+import Control from 'react-leaflet-control'
+import { Browser } from 'leaflet'
+import {ReactChoropleth, ReactDot, Choropleth, Dot} from './lib'
+import {createGrid} from 'browsochrones'
+import { render } from 'react-dom'
+
+const DOT = 'dot'
+const CHOROPLETH_QUANTILE = 'choropleth quantile'
+const CHOROPLETH_EQUAL = 'choropleth equal'
+const HUDSON_STREET = [40.73535, -74.00630]
+
+export default class GridualizerExample extends Component {
+  state = {
+    type: DOT,
+    grid: null
+  }
+
+  selectDot = (e) => {
+    this.setState({ ...this.state, type: DOT })
+  }
+
+  selectChoroplethQuantile = (e) => {
+    this.setState({ ...this.state, type: CHOROPLETH_QUANTILE })
+  }
+
+  selectChoroplethEqual = (e) => {
+    this.setState({ ...this.state, type: CHOROPLETH_EQUAL })
+  }
+
+  async componentWillMount () {
+    const raw = await window.fetch('/example.grid').then(res => res.arrayBuffer())
+    this.setState({...this.state, grid: createGrid(raw)})
+  }
+
+  render () {
+    return <LeafletMap center={HUDSON_STREET} zoom={12}>
+      <TileLayer
+        url={Browser.retina
```

Comment:
The main intent here is to be able to work on the library/component without embedding/linking it into another NPM modules directory, with all the mess that entails. Since at this point it's only for use by us, yes we can assume retina.